### DISCUSSION TITLE
Trend a plot with millisecond resolution

### DISCUSSION THREAD

**Discussion by toreernt** [plotly/Plotly.NET#447] [Q&A]
First of all thanks for the good work in here :-) I am using Plotly in Blazor to show trends of data and this  has been a really good solution. I have now data with milliseconds resolution, and Plotly does not show the milliseconds portion and just a straight line. X-axis is specified as follows : 
`  XAxis = new List<XAxis>
            {
               new XAxis
               {                   
                   TickFormat = "%H:%M:%S.%L"// Format for milliseconds

And adding data as follows : 
x.Add(DateTime.ParseExact(value.TimeStamp, "yyyy-MM-dd HH:mm:ss.fff", System.Globalization.CultureInfo.InvariantCulture));

I can see that milliseconds part is set, however it is zero in the trend itself. 

Might be an obvious mistake but struggle to see it :-)

A small snippet attached showing the plot and content of the DateTime object with ms...
![Plotlyms](https://github.com/plotly/Plotly.NET/assets/76165358/1cd0feed-423b-42d3-aabc-56cb5a55d679)

**Comment by kMutagene** [comment]
hey @toreernt could you attach the html output you are showing here? this could be either a serialization problem, but hard to tell without the output
